cep-sigstore-predicate.md

```diff
@@ -0,0 +1,72 @@
+# CEP - Standardizing the v1 predicate for sigstore attestations
+
+<table>
+<tr><td> Title </td><td> Standardizing the v1 predicate for sigstore attestations </td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Wolf Vollprecht &lt;wolf@prefix.dev&gt;</td></tr>
+<tr><td> Created </td><td> Feb 18, 2025 </td></tr>
+<tr><td> Updated </td><td> Feb 18, 2025</td></tr>
+<tr><td> Discussion </td><td>  </td></tr>
+<tr><td> Implementation </td><td>  </td></tr>
+</table>
+
+## Abstract
+
+We want to standardize attestations for the conda ecosystem.
+
+### Sigstore Attestations
+
+Sigstore attestations are cryptographic statements about software artifacts that provide:
+
+- Authenticity: Proof of who created/signed the artifact
+- Integrity: Verification that the artifact hasn't been tampered with
+- Transparency: Public record of signatures in a tamper-evident log
+
+### Key Components
+
+- Predicates: JSON documents containing metadata about the signing event, using the `in-toto` format
+- Signatures: Cryptographic proofs made using ephemeral keys
+- Rekor: A tamper-evident log that stores attestations
+- Fulcio: A certificate authority that issues short-lived certificates
```

Comment:
Is the claim of this CEP to make `rekor` and `falcio` the only acceptable transparency log and certificate authority for the conda ecosystem?